test/gtest/conv_igemm_mlir.cpp

```diff
@@ -88,6 +88,7 @@ TEST_P(ConfigWithHalf, HalfTest)
     const auto& handle = get_handle();
     if((miopen::StartsWith(handle.GetDeviceName(), "gfx103") ||
         miopen::StartsWith(handle.GetDeviceName(), "gfx906")) &&
+       miopen::IsEnvvarValueEnabled("MIOPEN_TEST_MLIR") &&
```

Comment:
[future] I think that we need to make this feature (is MLIR enabled or not) discoverable via some internal MIOpen API call, e.g. `miopen::debug::IsMlirEnabled()` and then use it in the tests instead of the env var. Because this env var is an additional dependence of gtests from ctest.

@JehandadKhan What do you think?